kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -78,11 +78,14 @@ with DAG(
             - name: base
               image: {{ config.image }}
               imagePullPolicy: {{ config.image_pull_policy }}
-              command:
-                - "bash"
-                - "-c"
+              env:
+                - name: MLFLOW_RUN_ID
+                  value: {% raw %}{{ task_instance.xcom_pull(key="mlflow_run_id") }}{% endraw %}
               args:
-                - MLFLOW_RUN_ID={% raw %}{{ task_instance.xcom_pull(key="mlflow_run_id") }}{% endraw %} kedro run -e {{ env }} --node '{{ node.name | safe }}'
```

Comment:
`bash` and `kedro` were dropped - do we move to the assumption that image has entrpoint with `kedro` statement in place?

Reply:
Good point! It was a bug, we will need "kedro run" as it is CMD in the image (not entrypoint).
Bash it no longer needed, as env can be set on the pod level now

Reply:
Just to double check - pod has CMD ['kedro', 'run'] , so we drop command from definition ,but into args we put kedro run as well?

Reply:
It's restored now :-)